Makefile

```diff
@@ -145,7 +145,7 @@ check:
 
 .PHONY: test
 test:
-	@$(GO) test -v $(SOURCE_DIRS_GO) -coverprofile cover.out
+	@$(GO) test -short -v $(SOURCE_DIRS_GO) -coverprofile cover.out
```

Comment:
http://peter.bourgon.org/go-in-production/#testing-and-validation seems to offer a slight different approach with a little more flexability. though i'm not sure if it would just add more complexity with no help.

this https://stackoverflow.com/a/25970712 post seems to also use the `-short` although also offers flags.